mink/configuration.py

```diff
@@ -38,18 +38,35 @@ def __init__(
         self,
         model: mujoco.MjModel,
         q: Optional[np.ndarray] = None,
+        joint_names: Optional[Sequence[str]] = None,
     ):
         """Constructor.
 
         Args:
             model: Mujoco model.
             q: Configuration to initialize from. If None, the configuration is
                 initialized to the default configuration `qpos0`.
+            joint_names: List of joints to be controlled. If None (default), all
+                joints are used
         """
         self.model = model
         self.data = mujoco.MjData(model)
         self.update(q=q)
 
+        if joint_names is None:
+            dof_ids = np.arange(model.nv)
+        else:
+            dof_ids = []
+            for joint_name in joint_names:
+                joint_id = mujoco.mj_name2id(
+                    model, mujoco.mjtObj.mjOBJ_JOINT, joint_name
+                )
+                if joint_id == -1:
+                    raise exceptions.InvalidJointName(joint_name, model)
+                dof_ids.append(model.jnt_dofadr[joint_id])
```

Comment:
should this be something like 
```
dof_ids.append(model.jnt_dofadr[joint_id]:model.jnt_dofadr[joint_id] + model.jnt_dofnum[joint_id])
```
?

I mean, I don't know if you support ball or free joints...

Reply:
Good catch, thanks @yuvaltassa!